rollup.config.js

```diff
@@ -868,37 +868,44 @@ export default function rollup(options) {
   return builds;
 }
 
+async function triggerLiveReload(appDir) {
+  // tickle live reload by touching the server entry
+  let serverEntryPaths = [
+    "entry.server.tsx",
```

Comment:
Is `entry.server.ts` intentionally omitted from this list?

Reply:
Hm, unsure.  This matches current behavior after https://github.com/remix-run/remix/pull/3059.  Looks like we used to only check `jsx`/`tsx`but esbuild allows `JSX` inside `.js` files so we added JSX.  I'll check if `.ts` supports JSX syntax and add if so

Reply:
In either case, I think it'd be good to add a comment explaining that esbuild allows JSX in `.js` and that's why `.js` is included here.

Reply:
Agreed - added one for clarity

Reply:
People can technically write `React.createElement` manually to avoid JSX. I don't see a reason to not just list all possible options.